scipy/stats/_distn_infrastructure.py

```diff
@@ -2339,14 +2339,30 @@ def integ(x):
     def expect(self, func=None, args=(), loc=0, scale=1, lb=None, ub=None,
                conditional=False, **kwds):
         """Calculate expected value of a function with respect to the
-        distribution.
+        distribution by numerical integration.
 
         The expected value of a function ``f(x)`` with respect to a
         distribution ``dist`` is defined as::
 
-                    ubound
-            E[x] = Integral(f(x) * dist.pdf(x))
-                    lbound
+                    ub
+            E[f(x)] = Integral(f(x) * dist.pdf(x)),
+                    lb
+
+        where `ub` and `lb` are arguments and `x` has the `dist.pdf(x)` 
```

Comment:
You need double backticks for monospace: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Reply:
I guess I'm a little confused by this:
https://numpydoc.readthedocs.io/en/latest/format.html#common-rest-concepts
where it states: 

Use *italics*, **bold** and ``monospace`` if needed in any explanations (but not for variable names and doctest code or multi-line code). Variable, module, function, and class names should be written between single back-ticks (`numpy`).

I read the passage to mean that `ub` and `lb` are variable names. I guess the first sentence I quoted refers to inlined code *only* where you include a ">>>" at the start of the line, and in those cases to not quote the variables b/c they wouldn't be evaluated as code if wrapped w/backticks. I'll assume I misunderstood (most likely case) and switch to monospace.  So `ub` becomes ``ub`` and `dist.pdf(x)` becomes ``dist.pdf(x)``, I'll adjust the codes from the other comment above accordingly, thanks!